e2e/test/iothub/AuthenticationWithTokenRefreshDisposalTests.cs

```diff
@@ -71,15 +71,17 @@ private async Task ReuseAuthenticationMethod_SingleDevice(Client.TransportType t
 
             // Create an instance of the device client, send a test message and then close and dispose it.
             DeviceClient deviceClient = DeviceClient.Create(testDevice.IoTHubHostName, authenticationMethod, transport);
-            await deviceClient.SendEventAsync(new Client.Message()).ConfigureAwait(false);
+            using var message1 = new Client.Message();
```

Comment:
I wonder if `SendEventAsync` fails to dispose on exception. I'd hope so, because then the user will want to try to resend it. That means it should be conditionally disposed by the creator.